src/smooch.js

```diff
@@ -1,12 +1,22 @@
-import Smooch from './smooch';
-import WebhooksAPI from './api/webhooks';
+import { AppUsersAPI } from './api/appUsers';
+import { ConversationsAPI } from './api/conversations';
+import packageInfo from '../package.json';
 
-export default class SmoochNode extends Smooch {
-  constructor(serverURL) {
-    super(serverURL);
+const SERVER_URL = 'https://api.smooch.io/v1';
 
-    this.webhooks = new WebhooksAPI({
+export class Smooch {
+  constructor(serverURL = SERVER_URL) {
+    this.VERSION = packageInfo.version;
+    this.serverURL = serverURL;
+
+    this.appUsers = new AppUsersAPI({
```

Comment:
@alavers if you think of something better than this pattern... I'm not too excited about needing to pass this guy to the underlying API to make them work.. coupled too tight IMO